cmd/postgres_exporter/main.go

```diff
@@ -132,9 +137,17 @@ func main() {
 		w.Write(landingPage)                                       // nolint: errcheck
 	})
 
+	webCfg := ""
```

Comment:
Minor
```suggestion
	var webCfg string
```

Reply:
why do you think `var webCfg string` is better?

Reply:
We trying to follow this convention: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
ok, I don't mind following any guideline that you follow. Is it the only code style that you follow?
Can we have this documented somewhere?

Reply:
The style guide that pointed out, contradict other generic code writing guidelines. ie. prefer explicit to implicit 
I don't mind following any guideline, but it need to be automated, on both fronts - to verify and fix.

Reply:
I guess in general we follow these three guides:

1. https://go.dev/doc/effective_go
2. https://github.com/golang/go/wiki/CodeReviewComments
3. https://dave.cheney.net/practical-go/presentations/qcon-china.html

And no, we don't have clear guidelines, and it's a problem.